### DISCUSSION TITLE
custom or set default of rtmp port on Home Assistant integration?

### DISCUSSION THREAD

**Discussion by boyphongsakorn** [blakeblackshear/frigate#3504] [Ideas]
i install frigate on TrueNAS 
TrueNAS can't me to use RTMP port 1935 or below 9000 (if someone know how to do that tell me)
yes easy thing to do is folk hass integration repo and add custom port
but i not know how long to that work 
my idea is i want to custom rtmp port for UI in hass integration

and..... that my idea thank

**Comment by NickM-27** [comment]
You can use https://docs.frigate.video/integrations/home-assistant/#rtmp-url-template to change the PORT. That being said, that definitely does not sound correct, you should use the search function here as I believe other users previously had some similar issue and figured out how to change it
